log/log_test.go

```diff
@@ -56,6 +56,35 @@ func TestContextMissingValue(t *testing.T) {
 	}
 }
 
+func TestWithPrefixAndSuffix(t *testing.T) {
+	t.Parallel()
+	buf := &bytes.Buffer{}
+	logger := log.NewLogfmtLogger(buf)
```

Comment:
I would prefer this test didn't depend on one of the formatting loggers. `WithPrefix` and `WithSuffix` have specific semantics related to the order of values passed to other loggers, but don't have any control over what those loggers do with the values or if the order is honored.

Look at the other tests in this file to see how they capture the `keyvals` slice in a `[]interface{}` variable and then verify the order in that slice.